### DISCUSSION TITLE
Build and test for Linux/ARM64

### DISCUSSION THREAD

**Discussion by odidev** [apache/grails-core#13929] [General]
## Description  
For aarch64 platform, I have built " Groovy/Grails " from source.  
 
## Steps   
Following STEPS (https://grails.org/download.html)  were used for the building and installation: 
1. Install Java
```
sudo apt update
sudo apt install openjdk-11-jdk
```
2. Install SDKMAN!
```
curl -s "https://get.sdkman.io" | bash
source "$HOME/.sdkman/bin/sdkman-init.sh
```
3. Install a Specific Version of Grails For example (grails v4.0.0)
  ```
sdk install grails 4.0.0" 
``` 
## Outcome  
 To validate that the build was successful we ran “grails --version " command.  
All this information is also added to ARM Dashboard Software Ecosystem Dashboard for Arm (https://www.arm.com/developer-hub/ecosystem-dashboard/).

**Comment by jeffscottbrown** [comment]
> Install a Specific Version of Grails For example (grails v4.0.0)

Are you reporting trouble running Grails 4 on Linux/Arm64?

**Reply by jeffscottbrown** [reply]
I see similar reports in other repositories like https://github.com/spring-projects/spring-boot/issues/43156.

**Reply by jamesfredley** [reply]
Will have to wait on https://github.com/github/roadmap/issues/970 since we use the GitHub-hosted runners to generate the Graalvm based native binaries.
